lib/db/memory.js

```diff
@@ -215,19 +233,41 @@ MemoryStore.prototype = {
     ret.token = _token;
     return P.resolve(ret);
   },
-  getToken: function getToken(token) {
-    return P.resolve(this.tokens[unbuf(token)]);
+  getAccessToken: function getAccessToken(token) {
+    return P.resolve(clone(this.tokens[unbuf(token)]));
   },
-  removeToken: function removeToken(id) {
+  removeAccessToken: function removeAccessToken(id) {
     delete this.tokens[unbuf(id)];
     return P.resolve();
   },
+  generateRefreshToken: function generateRefreshToken(vals) {
+    var token = {};
+    token.clientId = vals.clientId;
+    token.userId = vals.userId;
+    token.email = vals.email;
+    token.scope = vals.scope;
+    token.createdAt = new Date();
+    var _token = unique.token();
+    var ret = clone(token);
```

Comment:
nit: group vars with the top `var token = {}`. rename `var token = {};` to something like `tokenData = {}` to avoid `_token` vs `token`